File: 프로그래머스/1/12931. 자릿수 더하기/자릿수 더하기.java
import java.util.*;

public class Solution {
    public int solution(int n) {
        int answer = 0;
 //자바는 while(1)문법이 안됨(int를 bool처럼 사용 불가능)
        // [실행] 버튼을 누르면 출력 값을 볼 수 있습니다.
        for(int i = 0; i<10; i++){
            answer+= n%10;
            n/=10;
        }
        System.out.println("Hello Java")
;
    
        return answer;
    }
}

File: 프로그래머스/1/12931. 자릿수 더하기/자릿수 더하기.py
def solution(n):
    answer = 0
# 파이썬은 들여쓰기 잘 하기
    # [실행] 버튼을 누르면 출력 값을 볼 수 있습니다.
    print('Hello Python')
    while n > 0:
        answer += n % 10
        n //= 10  # 정수 나눗셈

    return answer